lab_3_generate_by_ngrams/main.py

```diff
@@ -309,6 +465,17 @@ def get_next_token(self, sequence: tuple[int, ...]) -> Optional[list[tuple[int,
 
         In case of corrupt input arguments or methods used return None.
         """
+        if not isinstance(sequence, tuple) or len(sequence) == 0:
+            return None
+
+        tokens_dict = self._model.generate_next_token(sequence)
+        if tokens_dict is None:
+            return None
+        if not tokens_dict:
+            return []
+
+        return sorted([(token, float(probability)) for token, probability in tokens_dict.items()],
```

Comment:
why convert to float only once?

Reply:
I only return this tuple (token, float(probability))